Tests/HummingbirdTests/FileMiddlewareTests.swift

```diff
@@ -512,4 +512,33 @@ final class FileMiddlewareTests: XCTestCase {
             }
         }
     }
+
+    func testFilesWithNonLowercaseFileExtensions() async throws {
+        let router = Router()
+        router.middlewares.add(FileMiddleware("."))
+        let app = Application(responder: router.buildResponder())
+
+        let testedExtensions: [(String, UInt)] = [
+            ("jpg", #line),
+            ("JPG", #line),
+            ("JpG", #line),
+            ("JPeG", #line),
+            ("JPEG", #line),
+        ]
+
+        for (index, (testedExtension, line)) in testedExtensions.enumerated() {
+            let fileURL = URL(filePath: "\(#function)-\(index)")
```

Comment:
This isn't compiling under Linux 5.x. You can use `fileURLWithPath` instead. At least that is what I'm using above.

Reply:
Is there a docker image to easily test this against?

Reply:
> This isn't compiling under Linux 5.x. 

I just looked at the failed actions and realized that you meant Swift 5.x on linux... I wasn't sure how to get an image with linux kernel 5.x, especially given that containers share the kernel with the host 😅

Reply:
You can use any `swift:5.x` docker image to test. I tend to use a devcontainer in VSCode to test these kinda things.

Reply:
Gotcha. That's what I ended up doing too :)

As for devcontainers, I also heard you mention that in a podcast. I assume that's a VSCode extension or whatever they call it? I'm using Xcode for all this so far and, while I'm happy with it, I'm not against seeing what alternatives can offer.